src/conversions/conversion.ml

```diff
@@ -37,13 +37,21 @@ class base ?(audio = false) ?(video = false) ?(midi = false) ~converter
 
     method self_sync = source#self_sync
 
-    (* The tmp_frame is intended to be filled by the underlying
-     * source. Content untouched by the converter are replaced by
-     * by content from the calling frame. Touched content get their
-     * own layer. *)
-    val mutable tmp_frame = Frame.create source#kind
+    val mutable tmp_frame = None
```

Comment:
Why not using `lazy` values for these like we do for frame settings?

Reply:
I've thought of that but you can't: you need to access `self` to get the content type (`self#ctype`) and OCaml won't let you have access to it when initializing instance variables (even under a lazy, which is legit but I guess difficult to check).

Reply:
Makes sense indeed thks.